pkg/resource/acl/hooks.go

```diff
@@ -45,8 +45,8 @@ func (rm *resourceManager) validateACLNeedsUpdate(
 	return nil
 }
 
-// aclActive returns true when the status of the given ACL is set to `active`
-func (rm *resourceManager) aclActive(
+// isACLActive returns true when the status of the given ACL is set to `active`
```

Comment:
you haven't mentioned this change in your PR description
if this and other changes are due to using a newer version of `code-generator`, then just state that in the PR description.

Reply:
Added them into PR description~